### PR TITLE
Enable registration

### PR DESCRIPTION
**Pull request by TrevorEdwards**
- [x] bad pull request etiquette

### CODE REVIEW

File: public/css/dashboard.less [hunk 1]
```diff
@@ -5,7 +5,7 @@
 @sidebar-brand-font-size: 24pt;
 
 .page-title, h1, h2, h3, h4, h5, h6 {
-  text-transform: lowercase;
+  //text-transform: lowercase;
```
Comment (mahsu) [inline public/css/dashboard.less]: remove

File: public/css/login.less [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @import "_global";
 
 body {
-  padding-top: 80px;
+  //padding-top: 80px;
```
Comment (mahsu) [inline public/css/login.less]: remove

File: routes/auth.js [hunk 3]
```diff
@@ -454,7 +455,7 @@ module.exports = function (io) {
                                 questions: {
                                     q1: req.body.q1,
                                     q2: req.body.q2,
-                                    hardware: req.body.hardware.split(",")
+                                    //hardware: req.body.hardware.split(",")
```
Comment (mahsu) [inline routes/auth.js]: remove or add a todo to add back for fall event

File: views/_global/_registration.jade [hunk 4]
```diff
@@ -88,16 +88,17 @@ block append content
...
+                            .form-group
+                                .checkbox
+                                    label
+                                        input#CoC(type='checkbox', name='CoC')
+                                        | I agree to abide by the #[a(href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf") Major League Hacking Code of Conduct]
 
                     fieldset
                         block submit
-                            button#submit.btn.btn-primary(type="submit") submit
+                            button#submit.btn.btn-default(type="submit") submit
```
Comment (mahsu) [inline views/_global/_registration.jade]: what's the difference between primary and default?

File: views/_navbar.jade [hunk 5]
```diff
@@ -1,22 +1,41 @@
...
-                    li: a(href="/login") Login
+nav.navbar.navbar-default
+    .container-fluid
+        .navbar-header
+            a.navbar-brand(href='#')
+                img(alt='BigRed//Hacks', src='/img/icon/brh-icon-full.png', width=40, height=40)
+        .navbar-header
+            a.navbar-text(href='/') Home
+        if isUser
+            .navbar-header
```
Comment (mahsu) [inline views/_navbar.jade]: i think you can still use the colon notation here?

File: views/_global/_registration.jade (hunk 4 above)
Comment (TrevorEdwards) [inline views/_global/_registration.jade]: i think primary is a part of bootstrap itself, and we haven't really defined its scheme. default, however, makes it look like our splash page buttons which is nice.

**Comment by TrevorEdwards** [comment]
boop

**Pull request merged**